pkg/capture/GPClassify.go

```diff
@@ -13,34 +13,21 @@
 
 package capture
 
+type Direction uint8
+
 // Direction detection states
 const (
-	Unknown uint8 = iota
+	Unknown Direction = iota
 	DirectionRemains
 	DirectionReverts
+	DirectionMaybeRemains
+	DirectionMaybeReverts
 )
 
-/*
-const multicastRangeIPv4 [3]byte = [3]byte{244, 0, 0}
-const broadcastIPv4      [4]byte = [4]byte{255, 255, 255, 255}
-const ssdpAddressIPv4    [4]byte = [4]byte{239, 255, 255, 250}
-*/
-
-// Lookup table storing frequently used destination ports which fall outside of the service
-// port range 1-1023. These are explicit exceptions to the direction heuristic below
-var specialPortsLoookupTable = [65536]bool{
```

Comment:
Subsumed under `classifyByPorts()` by generically looking at ephemeral / privileged ports. The rationale is that a) we can't rely on these ports to _actually_ be used for what we think they are (so it's unreliable by definition) and b) this is not even close to complete (i.e. _if_ we were to have such logic we'd need to go all the way and maintain a list of as many known ports as possible.
For applications / systems adhering to IANA a generic logic just looking at port numbers should give at least the same quality of results, with much less pain / maintenance overhead / complexity...

Reply:
Agreed. And a pain to maintain in the first place. Which time tells: we didn’t and left it where it was 2014. So all good.

One thing I was missing/overlooking in the logic as a special case is DHCP (67 for server, 68 for clients) handling.

Reply:
I deliberately removed it, because it's yet another cases that's covered under the new general logic. As you say: server is 67, client is 68, both privileged, UDP, so they follow the logic of sport > dport and hence you end up with exactly what you need (plus, the request is a broadcast, which is detected by the new broadcast/multicast logic). But I like the idea of adding a dedicated test case for those. Done!